pibooth/controls/__init__.py

```diff
@@ -1,8 +1,13 @@
 # -*- coding: utf-8 -*-
 
+from pibooth.utils import LOGGER
+from gpiozero import Device, LEDBoard
 try:
     from RPi import GPIO
+    LOGGER.debug("Start on Raspberry pi")
 except ImportError:
     # GPIO library only existes on Raspberry Pi
-    from pibooth.controls.gpio import GpioMock
-    GPIO = GpioMock()
+    # Set the default pin factory to a mock factory
+    from gpiozero.pins.mock import MockFactory
+    Device.pin_factory = MockFactory()
+    LOGGER.debug("Programme not start on Raspberry pi use MockFactory() mode")
```

Comment:
Typo, correct "not start" to "not started"

Reply:
Configuration of GPIO should be done only one time.
I propose the do it in the file `booth.py`